src/savegame_json.cpp

```diff
@@ -2355,7 +2355,11 @@ void item::io( Archive &archive )
                 itype_id( "laptop" ), // a monster-death-drop item group had this listed with random charges
                 itype_id( "usb_drive" ), // same as laptop
                 itype_id( "pipe" ), // similar as above, but in terrain bash result
-
+                itype_id( "throwing_knife" ), // Throwing weapon stacks used to be represented with charges.
```

Comment:
Maybe we need to create an item group for all these items?

Reply:
Actually subsequent testing shows there's a really long list, so yes.